lib/rate-limiter.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Bluebird = require('bluebird');
+
+exports.limit = (request, client, defaultRate) => {
```

Comment:
I think this plugin is simple enough to remain in index, like we've done with other hapi plugins. https://github.com/lob/hapi-sanitize-payload/blob/master/lib/index.js

@robinjoseph08 what do you think?